examples/logger.json

```diff
@@ -0,0 +1,6 @@
+{
+    "maxSize": 5242880,
+    "logDirectory": "/.autolabjs",
```

Comment:
Since the actual directory is "~/.autolabjs", it is better to specify the relative pathname. It is UNIX convention to start absolute path names with a forward slash. All relative path names start without a slash.